code/bit_manipulation/src/gray_code/grayCode.cpp

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Make sure your code run as expected.

Reply:
![image](https://user-images.githubusercontent.com/72879445/139799384-d0146f44-8575-4959-91a0-e2d731785ea5.png)
The code does run as expected in 4ms runtime.